Assets/Shopify/Examples/Scripts/Panels/ProductsPanel.cs

```diff
@@ -47,9 +81,29 @@ public class ProductsPanel : MonoBehaviour {
             foreach (var productView in visibleProductViews) {
                 productView.Load();
             }
+				
+            IsScrolledToBottom = scrollOffset.y < 0;
+        }
+
+        private void Update() {
```

Comment:
Why do you need an Update method here? could you just invoke Fetch products from within `OnScrollRectPositionChanged`?